### ISSUE TITLE
steam wont update on sandboxie

### ISSUE THREAD

**Issue by Meweu** [sandboxie-plus/Sandboxie#1926]
### What happened?

steam wont update on sandboxie

### Download link

https://github.com/sandboxie-plus/Sandboxie

### To Reproduce

go to sanboxie
go to steam on it
update it

### Expected behavior

update steam

### What is your Windows edition and version?

10 home

### In which Windows account you have this problem?

Not relevant to my request.

### Please mention any installed security software

Avast One

### What version of Sandboxie are you running?

v1.0.22

### In which sandbox type you have this problem?

In a Standard isolation sandbox (yellow sandbox icon).

### Where is the program located?

The program is installed both inside and outside the sandbox.

### Can you reproduce this problem on an empty sandbox?

I can confirm it also on an empty sandbox.

### Did you previously enable some security policy settings outside Sandboxie?

no

**Comment by SiNONiMiTY** [comment]
I can't reproduce this, what is the exact error message? I have my own Steam sandbox and Steam is updating properly.

**Comment by Meweu** [comment]
![image](https://user-images.githubusercontent.com/70145891/171959634-a1946ba2-1698-4594-8273-6d605aaec8ba.png)

**Comment by SiNONiMiTY** [comment]
```
# Custom Settings
BreakoutProcess=firefox.exe
###
Enabled=y
BlockNetworkFiles=y
BorderColor=#00ffff,off,6
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=AutoRecoverIgnore
ConfigLevel=9
AllowNetworkAccess=<BlockNetAccess>,n
BoxNameTitle=n
ShowForRunIn=n
ClosePrintSpooler=y
CopyLimitKb=-1
ForceFolder=D:\Games\Steam
LeaderProcess=steam.exe
NeverDelete=y
ClosedFilePath=<BlockNetDevices>,InternetAccessDevices
OpenFilePath=D:\Games
```

**Comment by NewKidOnTheBlock** [comment]
@Meweu 
Does it work with a current version of Sanboxie now?
I've been running a steambox for ages. It's updating just fine.

**Comment by DavidXanatos** [comment]
> ![image](https://user-images.githubusercontent.com/20263103/174610547-9b770be7-471b-413f-94ab-e3fe0231cc62.png) I am experiencing this.

this error was observed with variouse chromium based apps from time to time and will be resolved in 1.7.3
